src/rawParser.d.ts

```diff
@@ -13,4 +13,5 @@ export declare function parse(input: string):
   | {
       type: 'error'
       position: number
+      error: string
```

Comment:
bike shedding: should this be `message`? 😅

Reply:
Now that you say that I definitely prefer it. I'll go rename it.